tools/astarte_dev_tool/lib/constants/system.ex

```diff
@@ -0,0 +1,54 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteDevTool.Constants.System do
+  defmacro __using__(_opts) do
+    quote do
+      @command "docker"
+      @command_up_args [
+        "compose",
+        "-f",
+        "docker-compose.yml",
+        "-f",
+        "docker-compose.dev.yml",
+        "up",
+        "--build",
+        "--watch",
+        "-d"
+      ]
```

Comment:
The [`~w` sigil](https://hexdocs.pm/elixir/sigils.html#word-lists) might be handy here.

```suggestion
      @command_up_args ~w(compose -f docker-compose.yml -f docker-compose.dev.yml up --build --watch -d)
```